fvm/src/lib.rs

```diff
@@ -127,6 +127,8 @@ mod test {
             bs.put_cbor(&manifest, Code::Blake2b256).unwrap()
         };
 
+        let actors_cid = bs.put_cbor(&(0, manifest_cid), Code::Blake2b256).unwrap();
```

Comment:
wherent we supposed to eliminate v0? The original pr fixed this to use the v1 structure.

Reply:
This was just a test but I'm happy to get rid of it.